### PR TITLE
Update STAR lbs & render

### PR DESCRIPTION
**Pull request by wendaizhou**
* fix star lbs
* add demo/render_star to visualize star

### CODE REVIEW

File: mmhuman3d/models/body_models/star.py [hunk 1]
```diff
@@ -259,35 +258,65 @@ def forward(self,
...
+        if len(body_pose.shape) == 4:
+            # the shape of body pose is rot matrix, convert to angle_axis
+            body_pose = rotation_matrix_to_angle_axis(
+                body_pose.view(-1, 3, 3)).view(batch_size, -1, 3)
+
+        pose_quat = self.quat_feat(body_pose.view(-1, 3)).view(batch_size, -1)
         pose_feat = torch.cat((pose_quat[:, 4:], beta[:, 1]), 1)
 
-        R = aa_to_rotmat(body_pose.view(-1, 3)).view(batch_size, 24, 3, 3)
+        R = self.rodrigues(body_pose.view(-1, 3)).view(batch_size, 24, 3, 3)
```
Comment (caizhongang) [inline mmhuman3d/models/body_models/star.py:269]: Is there any reason why we are not using `aa_to_rotmat`?

Comment (wendaizhou) [reply mmhuman3d/models/body_models/star.py:269]: I've only verified the modified code by visualization. And I'm not sure if `aa_to_rotmat` and `self.rodrigues` can get the same result.  I'll confirm it later and change it back if there's no problem.

Comment (wendaizhou) [reply mmhuman3d/models/body_models/star.py]: > Is there any reason why we are not using `aa_to_rotmat`?

Sorry. After rechecking the file, there is indeed no difference between the two functions `aa_to_rotmat` and `self.rodrigues`, I have modified the code and recommitted it.

**Review by caizhongang — APPROVED** [review]
Looks good to me. Please do the following and we can merge this PR:
- Fix the linting problem.  Run the following commands under the repo directory:
    - `pip install pre-commit`
    - `pre-commit run --all-files`
    - git commit any changes.
- Fix the test problem. 
    - we have fixed a problem in the main branch, please merge the main branch into yours and the problem should go away.

@ttxskk please also take a look.

**Comment by wendaizhou** [comment]
Hi, the code has been updated to fix these two problems.😊

**Comment by caizhongang** [comment]
> Hi, the code has been updated to fix these two problems.😊

Thanks for the fix. There's some additional problem in the [build test](https://github.com/open-mmlab/mmhuman3d/actions/runs/5446539583/jobs/9907425312?pr=355):

```
=========================== short test summary info ============================
FAILED tests/test_models/test_body_models/test_star.py::test_star_specify_other_keypoint_mappings - AttributeError: 'NoneType' object has no attribute 'device'
FAILED tests/test_models/test_body_models/test_star.py::test_star_create_default_parameters - AttributeError: 'NoneType' object has no attribute 'device'
FAILED tests/test_models/test_body_models/test_star.py::test_star_init_parameters - AttributeError: 'NoneType' object has no attribute 'device'
FAILED tests/test_models/test_body_models/test_star.py::test_star_init_parameters_numpy - AttributeError: 'NoneType' object has no attribute 'device'
FAILED tests/test_models/test_body_models/test_star.py::test_star_forward_parameters - RuntimeError: shape '[1, 24, 3, 3]' is invalid for input of size 225
FAILED tests/test_models/test_body_models/test_star.py::test_star_init_and_forward_parameters - RuntimeError: shape '[1, 24, 3, 3]' is invalid for input of size 225
```

To run the test locally, you may set up the test files:
```
mkdir mmhuman3d_download
cd mmhuman3d_download
wget -O mmhuman3d.7z -q https://openmmlab-share.oss-cn-hangzhou.aliyuncs.com/mmhuman3d/mmhuman3d.7z
7za x mmhuman3d.7z
ls -l
cd ..
cp -r mmhuman3d_download/mmhuman3d/* ./
rm -rf mmhuman3d_download
```
and then run pytest:
```
pytest tests/test_models/test_body_models/test_star.py
```

**Comment by wendaizhou** [comment]
Hi, thanks for the guidance. The bug mentioned in `build test` has been fixed, but I'm not sure if `codecov` in failing checks needs to be handled.

**Comment by caizhongang** [comment]
> Hi, thanks for the guidance. The bug mentioned in `build test` has been fixed, but I'm not sure if `codecov` in failing checks needs to be handled.

>80% is fine. This PR looks good to me.

**Pull request merged**